application-analytics-ui/src/main/resources/Analytics/Code/Macros/VelocityMacros.xml

```diff
@@ -190,6 +190,25 @@
   #set ($macroId = $macro + '_' + $analyticsId)
 #end
 
+#macro(displayAnalyticsMacroTitle, $macroReferencer, $descriptionTranslationKey, $nameTranslationKey)
+  #set ($discard = $xwiki.ssx.use('Analytics.Code.AnalyticsJS'))
+  #set ($url = $xwiki.getURL($macroReferencer))
+  #set ($isDashboard = $doc.getObject('XWiki.GadgetClass'))
+  #set ($description = $escapetool.xml($services.localization.render($descriptionTranslationKey)))
+  #if ($isDashboard)
+    ## inside a dashboard so we need to add the eye icon to the tile of the dashboard gadget.
+    {{html}}
+      &lt;div class="hidden analytics-description-hidden" data-url=$url data-description="$description"&gt;&lt;/div&gt;
```

Comment:
* the ``hidden`` attribute is preferred over the ``.hidden`` CSS class
* the class name should be semantic; why not use simply ``analytics-description``?